traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -1387,7 +1387,7 @@ sub is_deliveryservice_valid {
 			active				=> [ is_required("is required") ],
 			cdnId				=> [ is_required("is required"), \&is_valid_int_or_undef ],
 			ccrDnsTtl			=> [ \&is_valid_int_or_undef ],
-			deepCachingType      => [ is_like( qr/^NEVER|ALWAYS$/, "must be NEVER or ALWAYS" ) ],
+			deepCachingType      => [ is_like( qr/^(NEVER|ALWAYS)$/, "must be NEVER or ALWAYS" ) ],
```

Comment:
really minor nit..   looks like current code is using TABs to align the arrows..  can you match?

Reply:
Done, but if it were up to me everything would be using spaces :)